Sources/PT.PM.Dsl.Tests/Data/ImproperValidationEmptyMethod.ptpm

```diff
@@ -1,3 +0,0 @@
-class : <[X509TrustManager|SSLSocketFactory]> {
-	<{ <[.+]>() {} }>
```

Comment:
I think we should use at least `<[\w+]>` syntax for identifiers especially for Java. This improves readability of identifiers and strings.